src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.jsx

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
 import {
-  BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip,
+  BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Label,
 } from 'recharts';
 import PropTypes from 'prop-types';
 
-// TODO - add legend, auto-scaling, sorting, etc
+// TODO - improve tickGap - e.g. the minTickGap={50} below needs to be dynamically calculated
 const Histogram = ({
   data,
   xAxisDataKey,
   barDataKey,
   chartWidth,
   chartHeight,
   barColor,
+  xAxisLegend,
 }) => (
   <BarChart
     width={chartWidth}
     height={chartHeight}
     data={data}
+    margin={{ top: 20, bottom: 65, right: 60 }}
   >
-    <XAxis dataKey={xAxisDataKey} />
+    <XAxis dataKey={xAxisDataKey} minTickGap={50} tickFormatter={(tick) => Math.round(tick * 10) / 10}>
+      <Label value={xAxisLegend || xAxisDataKey} position='bottom' offset={20} />
+    </XAxis>
     <YAxis />
     <Tooltip />
```

Comment:
I'm seeing some long numbers in the tooltip. Consider adding commas and shortening to whatever decimal place will be useful for the end user (2 probably?). 

Using Recharts Tooltip Formatter: https://stackoverflow.com/questions/51382831/recharts-add-commas-to-number-on-tooltip-on-bar-chart
Overriding Tooltip with React Component: https://recharts.org/en-US/examples/CustomContentOfTooltip
<img width="879" alt="image" src="https://user-images.githubusercontent.com/113449836/208769895-8dd6b7d0-a26a-48dc-ae95-4aa8c0ed024b.png">

Reply:
thanks for the tooltip tips! I'll look into it

Reply:
solved. 
![Screenshot 2022-12-21 at 15 01 58](https://user-images.githubusercontent.com/2900303/208923275-be1b3bf0-b422-4574-aa64-2d156013a0d0.png)